### ISSUE TITLE
Using models fine-tuned with huggingface/transformers

### ISSUE THREAD

**Issue by felicitywang** [UKPLab/sentence-transformers#71]
Has anyone tried loading a model fine-tuned by huggingface/transformers(their newest repo instead of the pytorch-transformers used in this repo)? I tried loading a RoBERTa model fine-tuned on sts-b/quora data using the newest `transformers` repository and then fine-tune the sentence-transformers with sts-b, and got really poor results. Not sure if it's a code/architecture-consistency problem. 
![image](https://user-images.githubusercontent.com/10904994/70184612-08b55700-169d-11ea-8b6c-a187cb886af9.png)

When I tried using the original roberta-base/roberta-large etc. models provided by huggingface, the results look normal.

Many thanks.

**Comment by nreimers** [comment]
Hi @felicitywang 
I haven't tested it. 

But currently I'm working on a new version, which will upgrade huggingface transformers to their newest version. I hope I can release it this or next week (if there are not unexpected issues with the version upgrade).

Best
Nils Reimers

**Comment by nreimers** [comment]
Hi @felicitywang 
I updated the code base in the v2.0.4 branch to the latest transformer version from huggingface:
https://github.com/UKPLab/sentence-transformers/tree/v0.2.4

It works as far as I can see without issues.

Note, BERT is sometime prone to some faulty gradient which "destroys" the complete model (performance drops to really low scores). A dirty patch is to use a different random seed and hoping that such a destructive gradient will not happen with a new random seed.

But it is quite common with small datasets that BERT (esp. BERT-large) produces really low scores due to the described issue with the gradients. 


Best
Nils

**Comment by felicitywang** [comment]
Thanks @nreimers ! This works great! I'll also try different random seeds, but for now the default one looks all right. 

Closing.
